frontend/src/components/collaborator/CollaboratorForm.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <div>
+    <e-text-field
+      v-if="status"
+      class="ec-status-field"
+      :value="translatedStatus"
+      readonly
+      :name="$tc('entity.campCollaboration.fields.status')"
+    >
+      <template #append>
+        <slot name="statusChange" />
+      </template>
+    </e-text-field>
+    <v-tooltip v-if="readonlyRole" eager bottom>
+      <span id="readonly">
+        {{ $tc('components.collaborator.collaboratorForm.roleHint') }}
+      </span>
+      <template #activator="{ on }">
+        <div tabindex="0" class="mt-3" v-on="on">
+          <e-select
+            v-model="localCollaboration.role"
+            fieldname="role"
+            readonly
+            aria-readonly="true"
+            aria-describedby="readonly"
+            :name="$tc('entity.campCollaboration.fields.role')"
+            :items="items"
+            :hint="$tc('components.collaborator.collaboratorForm.roleHint')"
```

Comment:
I think we could remove this copy of the hint. It is shown anyways in the tooltip when it's relevant. Also, right now, it's shown also on the collaborator create form, where it does not really make sense (I mean, it's good to know, but not really relevant there).

Reply:
#3499